boards/shields/x_nucleo_idb05a1/doc/index.rst

```diff
@@ -0,0 +1,82 @@
+.. _x-nucleo-idb05a1:
+
+X-NUCLEO-IDB05A1: Bluetooth Low Energy expansion board based on the SPBTLE-RF
+module
+#####################################################################
```

Comment:
I guess this title syntax should be fixed

Reply:
If you are refering to "SPBTLE-RF" and "module" words being split on two lines, this is done to comply with "80 character max per line" rule

Reply:
Yes, but I think this won't work with rst or html syntax

Reply:
Tried with rst2pdf, indeed this break the title, nice catch thank you !